app/presenters/summary/sections/payment_details.rb

```diff
@@ -22,7 +22,7 @@ def answers # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/Per
               if payment_details.nil?
                 Components::FreeTextAnswer.new(
                   formatted_payment_name,
-                  I18n.t('summary.does_not_get'),
+                  type_suffix.include?('outgoing') ? I18n.t('summary.does_not_pay') : I18n.t('summary.does_not_get'),
```

Comment:
We could have a method that gets the text in each of the child classes like `formatted_payment_name` but not blocking

Reply:
Yeah I thought about something like this but thought the whole answers method could probably be refactored (I don't disagree with rubocop in this case 😅) but didn't want to go down that rabbit hole!